src/containers/Pages/AboutPage/AboutPage.module.scss

```diff
@@ -0,0 +1,126 @@
+@import "../../../assets/designs/commonTheme.scss";
+
+.content {
+  background-color: $backgroundColor;
+  height: fit-content;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+p {
+  text-align: justify;
+  text-justify: inter-word;
+  color: #5f5f5f;
+  letter-spacing: -0.4px;
+}
+.aboutUs {
+  display: flex;
+  flex-direction: column;
+  align-self: center;
+  width: 50%;
+  letter-spacing: -0.4px;
+  color: #5f5f5f;
+  h1 {
+    align-self: center;
+  }
+  h2 {
+    align-self: center;
+    letter-spacing: -1.02px;
+  }
+}
+.team {
+  width: 65%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-self: center;
+  justify-content: center;
+}
+
+.magnifyGlassContainer {
+  margin: 1vw;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: flex-start;
+  align-content: flex-start;
+  width: 17vw;
+  height: 500px;
+  z-index: 80;
+  &:hover .imgAvatar {
+    filter: blur(4px);
+    -webkit-filter: blur(4px);
+  }
+  &:hover .magnifyGlassHandle {
+    filter: blur(1px);
+    -webkit-filter: blur(1px);
+  }
+  &:hover .magnifyGlass .logoAvatarContainer .avatarIcon {
+    visibility: visible;
+  }
+  .magnifyGlass {
+    align-self: center;
+    position: relative;
+    border: 0.5vw solid white;
+    border-radius: 50%;
+    width: 15vw;
+    height: 15vw;
+    z-index: 86;
+    .imgAvatar {
+      border-radius: 50%;
+      height: 100%;
+      width: 100%;
+      z-index: 90;
+    }
+
+    .logoAvatarContainer {
+      align-self: center;
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      position: absolute;
+      top: 0%;
+      justify-content: space-evenly;
+      width: 100%;
+      height: 100%;
+      .avatarIcon {
+        height: 3.5vw;
+        width: 3.5vw;
+        z-index: 100;
+        align-self: center;
+        visibility: hidden;
+      }
+    }
+  }
+  .magnifyGlassHandle {
+    height: 0.5vw;
+    width: 25%;
+    background-color: white;
+    transform: rotate(50deg);
+    position: absolute;
+    z-index: 0;
+    left: 75%;
+    border-radius: 15%;
+  }
+
+  &:hover .magnifyGlass {
+    transition: border-width 0.2s ease-in-out;
+    border: 0.7vw solid white;
+  }
+  &:hover .magnifyGlassHandle {
+    transition: border-width 0.2s ease-in-out;
+    border: 0.1vw solid white;
+  }
```

Comment:
there is a duplicate css class which seem to server the same purpose, was this intended. its better to keep all things in the same container, unless it serves a purpose

Reply:
reason this is different is because if I set the handle border change to the same as the circle, it becomes disproportionately larger